src/histograms/utils/HistogramUtils.ts

```diff
@@ -287,42 +288,42 @@ export class HistogramUtils {
           if (dateInterval !== undefined && dateInterval !== null && dateInterval > 0 && dateInterval < 1) {
             roundPrecision = this.getRoundPrecision(dateInterval);
           }
-          return this.round(value, roundPrecision).toString();
+          return formatWithSpace(this.round(value, roundPrecision));
         }
       }
     }
   }
 
   public static getFormatFromDateInterval(dateInterval): string {
     const duration: moment.Duration = moment.duration(dateInterval);
-    let format;
+    let f: string;
```

Comment:
You could also not use any variable and return the string from each `case`(it will just remove some lines)

Reply:
Fixed